webapp/components/survey/Record/Record.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import SurveyFormView from '@webapp/components/survey/SurveyForm'
```

Comment:
I would call SurveyForm as well.. sorry missed that before

Reply:
sure, my mistake